t/50mruby-http-request.t

```diff
@@ -542,6 +542,7 @@ EOT
 };
 
 subtest 'timeout' => sub {
+    my $blackhole = t::Util::find_blackhole_ip(443);
```

Comment:
Maybe you want to export `find_blackhole_ip` and use it without the package name?